### ISSUE TITLE
Prevent Code block from showing up in the Mind Map as a subpoint

### ISSUE THREAD

**Issue by nickhir** [lynchjames/obsidian-mind-map#32]
Hello,
I was wondering if there is a option to prevent code blocks from showing up in the Mind map as subpoints.

Cheers!

**Comment by andrezgz** [comment]
I agree with this! They consume a lot of space within the mind map. Same problem when rendering blockquotes. Both should be added only when a certain configuration option is active.

**Comment by an2018x** [comment]
It can be hidden by editing .obsidian/plugins/obsidian-mind-map/main.js file. You can search it using everything.
then and a new line return ''; after the function rules.fence = function(tokens, idx, options, env, instance) {
just like this:
![image](https://user-images.githubusercontent.com/44701394/197399470-9cb3b178-3e87-4abf-8922-9e25e398d61d.png)
and below is the render result after hiding the code blocks in mind map:
![image](https://user-images.githubusercontent.com/44701394/197399511-4bda3b08-e5f8-476f-8f24-c75ce81f1863.png)
And I agress with @andrezgz  that it should be added as a configurable option considering that it not beautiful when rendering code blocks in mindmap.
